utils/pip/pipdeptree/scriptCreator.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+)
+
+const (
+	pipDepTreeVersion             = "3"
+	pipDepTreeContentFileName     = "deptreescript.go"
+	pipDepTreePythonScript        = "pipdeptree.py"
+	pipDepTreeContentRelativePath = "utils/pip"
+)
+
+// Run manually  to generates new pipDepTreeContentFileName from pipDepTreePythonScript if there is new one and update pipDepTreeVersion number
+func main() {
+	wd, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+
+	// Check if a content file of the latest version already exists
```

Comment:
// Check if a content file of the latest version already exists
-->
// Check if a pip-dep-tree file of the latest version already exists